lib/grizzly/zwave/command_classes.ex

```diff
@@ -367,7 +367,7 @@ defmodule Grizzly.ZWave.CommandClasses do
 
   def maybe_concat_command_classes(binary, _, <<>>), do: binary
 
-  def maybe_concat_command_classes(binary, :non_secure_controlled_bin, ccs_bin),
+  def maybe_concat_command_classes(binary, :non_secure_controlled, ccs_bin),
```

Comment:
This doesn't look like a tracing change?

Reply:
Correct, I can pull that out into another PR, it was just breaking the tracer's ability to get some data out from the binary.